src/components/Card/Card.js

```diff
@@ -1,28 +1,92 @@
 import { useState } from 'react';
 
+import { MdEdit, MdSave, MdEditOff } from 'react-icons/md';
+
 import './Card.css';
 
 function Card(props) {
+  //checkbox
   const [isChecked, setIsChecked] = useState(false);
 
+  //edit
+  const [isEditing, setIsEditing] = useState(false);
+
+  const [title, setTitle] = useState(props.title);
```

Comment:
Could you please do not store here a copy of props.title and descr and just use it in your component. When save action fired - change the data of App component.